lib/components/src/typography/shared.tsx

```diff
@@ -20,8 +20,9 @@ export const headerCommon = ({ theme }: { theme: Theme }): CSSObject => ({
 });
 
 export const codeCommon = ({ theme }: { theme: Theme }): CSSObject => ({
+  lineHeight: 1,
```

Comment:
This applies to all code blocks. Can you review the chromatic changes? I see 4 for proprow and then another one that looks like it might be a bug